### ISSUE TITLE
AppSwitch does not provide any keyboard accessibility

### ISSUE THREAD

**Issue by roastery-zz** [coreui/coreui-react#44]
This code does not work as expected:

```jsx
<AppSwitch tabIndex="0" variant="pill" color="primary"
  checked={isChecked} onChange={this.handleChange}/>
```

Actual result:
* tabIndex is rendered on the `<input>` which has `display: none`.
* This does not work in any browser I've tested, the control is skipped entirely in the tab order.

Desired result:
* tabIndex should be rendered on the wrapped `<label>`.
* Keyboard support per [ARIA checkbox guidelines](https://www.w3.org/TR/wai-aria-practices/#checkbox) should be implemented (pressing Space or Enter should toggle the switch)
